package.json

```diff
@@ -39,15 +39,16 @@
     "@babel/preset-react": "^7.12.0",
     "babel-loader": "^8.2.0",
     "babel-preset-gatsby-package": "^0.12.0",
-    "dtslint": "^4.0.0",
-    "eslint": "^7.13.0",
-    "eslint-config-prettier": "^7.0.0",
-    "eslint-config-xo": "^0.33.0",
+    "dtslint": "^4.0.8",
+    "eslint": "^7.23.0",
```

Comment:
these updates appear to already be/have been in semver range?

Reply:
When debug an issue related to duplicate versions of `@typescript-eslint/eslint-plugin` due to `gatsby`, I tried to upgrade them to latest versions manually.